spec/presenters/tree_builder_images_spec.rb

```diff
@@ -0,0 +1,48 @@
+describe TreeBuilderImages do
+  context 'TreeBuilderImages' do
```

Comment:
Also, it tends to work better if the top level is focused on the method you are calling.